emoji-ios/src/main/java/com/vanniktech/emoji/ios/IOSEmojiProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.vanniktech.emoji.ios;
+
+import android.support.annotation.NonNull;
+
+import com.vanniktech.emoji.EmojiProvider;
+import com.vanniktech.emoji.emoji.EmojiCategory;
+import com.vanniktech.emoji.ios.category.ActivityCategory;
+import com.vanniktech.emoji.ios.category.FlagsCategory;
+import com.vanniktech.emoji.ios.category.FoodsCategory;
+import com.vanniktech.emoji.ios.category.NatureCategory;
+import com.vanniktech.emoji.ios.category.ObjectsCategory;
+import com.vanniktech.emoji.ios.category.PeopleCategory;
+import com.vanniktech.emoji.ios.category.PlacesCategory;
+import com.vanniktech.emoji.ios.category.SymbolsCategory;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class IOSEmojiProvider implements EmojiProvider {
+    @Override
+    @NonNull
+    public Map<String, EmojiCategory> getCategories() {
+        final LinkedHashMap<String, EmojiCategory> result = new LinkedHashMap<>();
+
+        result.put("People", new PeopleCategory());
```

Comment:
Hmm, they are not really used anywhere, I think it's best to make `getCategories` just return a `List`.